changelogs/fragments/3926-fix-gitlab-project-avatar-path.yml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - gitlab - gitlab_project needs to check for avatar_path before trying to open (https://github.com/ansible-collections/community.general/issues/3926).
```

Comment:
I just saw that `avatar_path` hasn't been in a release yet, so this doesn't need a changelog fragment. Please remove this file.
```suggestion
```